modules/combat.py

```diff
@@ -1,11 +1,15 @@
 import math
 import string
+from threading import Thread
+from typing import List, Any
+
+from scipy import spatial
 from util.logger import Logger
 from util.utils import Region, Utils
-from scipy import spatial
-from threading import Thread
+
 
 class CombatModule(object):
+    l: List[Any]
```

Comment:
Subjective changes on the order and I don't comprehend why would you import `List` neither the last line is for. 
Sorry for my poor knowledge about the topic.